test/functions/typeMethods/arg-type-type-method.bad

```diff
@@ -1,2 +1,2 @@
-arg-type-type-method.chpl:6: In function 'test':
```

Comment:
It'd be nice to do this in a separate PR, but it's good that you call it out in the PR message.  Should it also be mentioned in the main subject line?  I think you have space to do so